lib/webdriver_script_adapter/execute_async_script_adapter.rb

```diff
@@ -51,7 +51,7 @@ def async_results_identifier
     end
 
     def callback(resultsIdentifier)
-      "function(returnValue){ #{resultsIdentifier} = returnValue; }"
+      "function(err, returnValue){ #{resultsIdentifier} = (err || returnValue); }"
```

Comment:
Probably worth improving this to know when the result is an error vs a return. (Probably not on this PR, but separately?) Perhaps assigning an object of the form `{err, value}`?